code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -264,3 +264,27 @@
 	id = /datum/reagent/medicine/psicodine
 	results = list(/datum/reagent/medicine/psicodine = 5)
 	required_reagents = list( /datum/reagent/medicine/mannitol = 2, /datum/reagent/water = 2, /datum/reagent/impedrezene = 1)
+
+/datum/chemical_reaction/burnmix // Chemistry recipe
+	name = "BurnMix"
+	required_temp = 700
+	id = /datum/reagent/medicine/burnmix
+	mix_message = "The solution boils, binding the remaining ingredients."
+	results = list(/datum/reagent/medicine/burnmix = 2)
+	required_reagents = list(/datum/reagent/fuel = 1, /datum/reagent/carbon = 1, /datum/reagent/iron = 1)
+
+/datum/chemical_reaction/burnmix2 // Ghetto recipe
+	name = "BurnMix"
+	id = /datum/reagent/medicine/burnmix
+	required_temp = 500
+	mix_message = "The solution bubbles, binding the ingredients."
+	results = list(/datum/reagent/medicine/burnmix = 3) // Fuel burns away to form BurnMix from Nutriment and ash
+	required_reagents = list(/datum/reagent/fuel = 1, /datum/reagent/consumable/nutriment = 1, /datum/reagent/ash = 2) // Some welding fluid, any food and ash (burn some paper)
+
+/datum/chemical_reaction/burnmix3 // Ghetto recipe 2; wasteful . Replaces nutriment with ethenol but most of the ethenol dissolves in the process
+	name = "BurnMix"
+	id = /datum/reagent/medicine/burnmix
+	required_temp = 500
+	mix_message = "The solution partially dissolves, binding the remaining ingredients."
+	results = list(/datum/reagent/medicine/burnmix = 3) // fuel and most ethenol burns away to form BurnMix from remaining ethenol and ash
+	required_reagents = list(/datum/reagent/fuel = 1, /datum/reagent/consumable/ethanol/beer = 3, /datum/reagent/ash = 2) // Take a booze bottle, drink some , add welding fuel and ash. Heat up with lighter
```

Comment:
Hello Alex,
I get what you are trying to do but our codebase does not currently support using the "tree" as a crafting recipe. 
What you have done here is made the recipe chem only as it will take your ethanol requirement literally as in requiring that "ethanol" selection in the chem dispenser. 

I was brainstorming with Altoids earlier and he managed to find the reason for this. The reason for this functionality not existing is located inside of Yogstation-TG\code\modules\reagents\chemistry\holder.dm  . Specifically line if (R.type == reagent)
It does an exact compare here meaning that it will NOT accept any alchohol anymore. Using space beer is my temporary workaround for BurnMix until I fix our codebase to be able to accommodate tree crafting (which is another project on its own)

Your proposed change does not fit the vision I had for Burnmix as I want the recipe that you modified to be a "ghetto" recipe able to be made outside of chem. I request this change to be denied.

Reply:
Oh that's dumb

Reply:
Yeah hella dumb. Hopefully I get around to fixing it sometime because I kind of want BurnMix to use any alchohol